### DISCUSSION TITLE
How to configure globally the showDelay option of v-tooltip?

### DISCUSSION THREAD

**Discussion by matheus-sunderhus** [primefaces/community#318] [PrimeVue]
Hi, I'm facing a problem to solve this specific case, how to globally configure to all tooltips had a default showDelay of 300, to avoid passing this on each v-tooltip directive in my code.

I'm currently have this in each component that uses a tooltip:
<button v-tooltip.bottom="{ value: tooltipText, showDelay: 200 }" />

I'm want to achieve something like this
// main.js file
createApp(App,{
  pt:{
    directives:{
     tooltip:{}// configure tooltip to have a default showDelay
     }
  }
}

I don't know if this is possible, or I'm just using the wrong path for it. Anyways, If someone knows how to configure globally the showDelay of tooltip will be awesome!

**Comment by tugcekucukoglu** [comment]
Hi,

It's on our roadmap that global configuration on directives.

Thanks for your consideration.

**Reply by sunderhus** [reply]
ty

**Comment by kuaj** [comment]
hey @tugcekucukoglu 

is this functionality still in the roadmap? 
https://primevue.org/roadmap/

thank you for your hard work!

**Comment by tednaaa** [comment]
any news on this guys?
